hw07_file_copying/main.go

```diff
@@ -18,5 +19,9 @@ func init() {
 
 func main() {
 	flag.Parse()
-	// Place your code here.
+	if err := Copy(from, to, offset, limit); err != nil {
+		fmt.Println("Error occurred while copying file")
```

Comment:
```suggestion
		fmt.Printf("Error occurred while copying file, %v", err)
```
не желательно мьютить ошибки возвращаемые уровнем ниже, т.к. это усложняет дебаг приложений